collectors/30/kafka_offset_collector.py

```diff
@@ -16,23 +21,23 @@ def format_tsd_key(metric_key, metric_value, timestamp, tags={}):
     output = '{} {} {} {}'.format(metric_key, timestamp, metric_value, expanded_tags)
     return output
 
-def format_kafka_broker_offset_tsd_key(topic, partition, offset):
+def format_kafka_broker_offset_tsd_key(env, topic, partition, offset):
     metric = 'kafka.broker.offset'
-    tags = dict(topic=topic, partition=partition)
+    tags = dict(env=env, topic=topic, partition=partition)
     timestamp = int(time.time())
     return format_tsd_key(metric, offset, timestamp, tags)
 
 
-def format_kafka_consumer_lag_tsd_key(consumer_group, topic, lag):
+def format_kafka_consumer_lag_tsd_key(env, consumer_group, topic, lag):
     metric = 'kafka.lag'
-    tags = dict(consumer_group=consumer_group, topic=topic)
+    tags = dict(env=env, consumer_group=consumer_group, topic=topic)
     timestamp = int(time.time())
     return format_tsd_key(metric, lag, timestamp, tags)
 
 
-def format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offset):
+def format_kafka_consumer_offset_tsd_key(env, consumer_group, topic, partition, offset):
     metric = 'kafka.consumer.offset'
-    tags = dict(consumer_group=consumer_group, topic=topic, partition=partition)
+    tags = dict(env=env, consumer_group=consumer_group, topic=topic, partition=partition)
     timestamp = int(time.time())
     return format_tsd_key(metric, offset, timestamp, tags)
```

Comment:
I think getting rid of these helper methods and calling `format_tsd_key` directly would be a good idea since they're all doing the same thing. 
It would also make it easier to see where each metric is being reported from. 
(They all have 1 caller each so there isn't any code reuse.)